source/geometries/NextDemoSiPMBoard.cc

```diff
@@ -127,12 +129,12 @@ void NextDemoSiPMBoard::Construct()
   G4Material* mother_gas = mother_phys_->GetLogicalVolume()->GetMaterial();
 
 
-  /// Board configuration checks
-  // Coating require membranes
-  if (coating_thickn_ > 0.)
-    if (membrane_thickn_ == 0.)
-      G4Exception("[NextDemoSiPMBoard]", "Construct()", FatalException,
-      "Coating require membranes");
+  // /// Board configuration checks
```

Comment:
I had to comment the lines to make the code compatible with Run 10. Since coating is placed over the membranes in Run 7, Run 8 and Run 9, this exception might be useful. I commented out this lines just to remember this fact.

In my opinion, since the membranes we are simulating are dummy ones (just a Xenon volume with no effect in the simulation) I don't even think they are useful in the geometry. But that is something @jmunozv might want to clarify.

Reply:
I see. If need to keep them (depending on what experts say), we could add lines to do this check depending on the run configuration used. I guess this will mean to add a config variable.

Reply:
What are we going to do with these commented lines? Are we going to keep them like that or adapt them to the case without membrane? Maybe we could include `hole_coated_` in the condition, like `if (!hole_coated_ && (membrane_thickn_ == 0.) )`